### PR TITLE
Documentation changes for 0.6.0 release

### PR DESCRIPTION
**Pull request by aaltay**
R: @davorbonaci @sb2nov

### CODE REVIEW

File: src/get-started/downloads.md [hunk 1]
```diff
@@ -49,6 +49,11 @@ Additionally, any API may change before the first stable release, i.e., between
 ## Releases
 
 ### 0.5.0
```
Comment (sb2nov) [inline src/get-started/downloads.md]: 0.6.0

Comment (aaltay) [inline src/get-started/downloads.md]: Good catch, thank you. Fixed.

**Comment by sb2nov** [comment]
LGTM

**Comment by aaltay** [comment]
Retest this please

**Pull request closed without merging**